### ISSUE TITLE
EFO OTAR Profile v3.34.0 contains a cycle with MONDO:0100321 and EFO:0000763

### ISSUE THREAD

**Issue by dhimmel** [EBISPOT/efo#1272]
`efo_otar_profile.owl` for [`v3.34.0`](https://github.com/EBISPOT/efo/releases/tag/v3.34.0) contains a cycle in its `rdfs:subClassOf` relationships, meaning that the ontology is not a directed acyclic graph.

`MONDO:0100321` is a subclass of `EFO:0000763` which is a subclass of `MONDO:0100321`. Note that this only showed up in the OTAR (Open Targets release) and not `efo.owl`.

Related to discussion at https://github.com/EBISPOT/efo/issues/961 regarding whether cycles can be detected prior to release.

**Comment by ireneisdoomed** [comment]
Hello Daniel,

this is Irene from the OT Data team. I have checked this issue to make sure it doesn't cause any problems downstream and I don't see such relationship.

I have inspected `efo_otar_profile.owl v.3.34.0` with [Protégé](https://protege.stanford.edu) and both terms only link to `EFO_0005741`:
![image](https://user-images.githubusercontent.com/45119610/134892125-c19a03d2-bd8d-40d6-98e5-349b8367ddd5.png)

**Comment by dhimmel** [comment]
Thanks @ireneisdoomed for your help! The two relevant terms here are:

- http://www.ebi.ac.uk/efo/EFO_0000763 "viral disease"
- http://purl.obolibrary.org/obo/MONDO_0100321 "viral disease or post-viral disorder"

I am not sure why Protege is giving different subclass relations than the raw OWL.

Looking at the OWL file we can see that `EFO_0000763` is a subClassOf `MONDO_0100321` with this shell command:

```sh
curl --silent --location https://github.com/EBISPOT/efo/releases/download/v3.34.0/efo_otar_profile.owl | grep --after-context=15 '<!-- http://www.ebi.ac.uk/efo/EFO_0000763 -->'
```
_some lines omitted_:

```xml
    <!-- http://www.ebi.ac.uk/efo/EFO_0000763 -->
    <Class rdf:about="http://www.ebi.ac.uk/efo/EFO_0000763">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MONDO_0100321"/>
        <rdfs:subClassOf rdf:resource="http://www.ebi.ac.uk/efo/EFO_0005741"/>
```

With the same query for `MONDO_0100321`, we see that `MONDO_0100321` is a subClassOf `EFO_0000763`:

```sh
curl --silent --location https://github.com/EBISPOT/efo/releases/download/v3.34.0/efo_otar_profile.owl | grep --after-context=15 '<!-- http://purl.obolibrary.org/obo/MONDO_0100321 -->'
```

```xml
    <!-- http://purl.obolibrary.org/obo/MONDO_0100321 -->
    <Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0100321">
        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MONDO_0100336"/>
        <rdfs:subClassOf rdf:resource="http://www.ebi.ac.uk/efo/EFO_0000763"/>
```

**Comment by dhimmel** [comment]
Hi @zoependlington thanks for taking on this assignment!

Any idea what the cause is and whether it could be fixed for the v3.35.0 release coming in a few days?

**Comment by zoependlington** [comment]
This should now be fixed in today's release.
